synapse/lib/layer.py

```diff
@@ -1912,10 +1912,10 @@ async def iterLayerNodeEdits(self):
                 nodeedits[2].append(edit)
                 continue
 
-            if flag == 1:
-                if not nodeedits[0] == buid:
-                    continue
+            if not nodeedits[0] == buid:
+                nodeedits = (buid, None, [])
```

Comment:
WIth a None form, i'm not sure the node edit will be consumable correct?

Reply:
Updated with form lookups and form None check in node set / tag set editors